src/styles/_catalog.scss

```
@@ -1,22 +1,35 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(1, $card-width);
```

let's remove this redundant `repeat()` function, one repeating is still 1 column
```suggestion
  grid-template-columns: $card-width;
```